### DISCUSSION TITLE
Two buildings to automate, two instances of zigbee2mqtt

### DISCUSSION THREAD

**Discussion by derekmurawsky** [Koenkk/zigbee2mqtt#5734] [General]
First off, thank you for the awesome system. I'm amazed how game-changing this is. 

I have one Home Assistant that I'm setting up to manage and control my homestead: a main house and a barn with my office/lab. I plan on having one zigbee2mqtt instance on a raspberry pi in each building sending to the MQTT broker in Home Assistant. Are there any caveats to be aware of with a setup like this? I'm specifically concerned that there might be some overlap with the `base_topic: zigbee2mqtt` or am I just over-thinking it?

Thanks,
-D

**Comment by TheNutHouse84** [comment]
Change the base topic on each pi to reflect each building. z2mhouse & z2mbarn
